invenio_search_ui/assets/js/invenio_react_search_ui/App.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2020 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import { Container, Grid } from "semantic-ui-react";
+import {
+  ReactSearchKit,
+  SearchBar,
+  BucketAggregation,
+  EmptyResults,
+  Error,
+  ResultsLoader,
+  withState
+} from "react-searchkit";
+import { Results } from "./Results";
+import { InvenioSearchApi } from "react-searchkit";
+
+import "semantic-ui-css/semantic.min.css";
+
+const OnResults = withState(Results);
+
+const resultsPerPageValues = [
+  {
+    text: "10",
+    value: 10
+  },
+  {
+    text: "20",
+    value: 20
+  },
+  {
+    text: "50",
+    value: 50
+  }
+];
+
+export class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { activeIndex: -1 };
+  }
+
+  render() {
+
+    const config = new InvenioSearchApi({
+      axios: {
+        url: this.props.config.api,
+        withCredentials: true
+      },
+      timeout: 5000,
+      headers: { Accept: this.props.config.mimetype, }
+    });
+
+    return (
+      <ReactSearchKit searchApi={config}>
+        <Container>
+          <Grid>
+            <Grid.Row>
+              <Grid.Column width={4} />
+              <Grid.Column width={12}>
+                <SearchBar />
+              </Grid.Column>
+            </Grid.Row>
+          </Grid>
+          <Grid relaxed padded>
+            <Grid.Row columns={2}>
+              <Grid.Column width={4}>
+                <Grid.Row><Grid.Column/></Grid.Row>
+                <BucketAggregation
```

Comment:
We should use `RECORDS_REST_FACETS` to render these buckets

Reply:
done, thanks for noticing this